61.md

```diff
@@ -0,0 +1,145 @@
+NIP-61
+======
+
+Event Sets
+----------
+
+`draft` `optional`
+
+An event set is represented by a group of events, not a single event.
+Each event of a specific set has the same pubkey and `n` (set "n"ame) tag.
+
+An event can have many `n` tags, thus being member of many event sets.
+To remove an event from a set, remove the corresponding `n` tag from the event or
+send a deletion event (`kind:5`).
+
+## Referencing
+
+An event set is referenced by an `s` tag. It
+references many events from an author instead of a single one:
+
+`["s", "<kind, optional>,<other kind, optional>:<32-bytes lowercase hex of a pubkey>:<n tag value>", "<recommended relay URL, optional>"]`
+
+One may also use an `nset` [NIP-19](19.md) entity through a [NIP-21](21.md) URI: `nostr:nset1qqstn...794234d`.
+
+## Set types
+
+There are two types of sets: standard and user-generated.
+
+### Standard
+
+Standard sets have one of the names (the `n` tag value) listed here to guarantee interoperability.
+The convention is to prefer lowercase English words in singular form with dash separator.
+
+| n-tag | description | expected d-tag prefixes if using kind:30161 |
+|-|-|-|
+| bookmark | uncategorized, "global" list of things a user wants to save | "i:" (kind:1 notes), "a:" (kind:30023 articles), "t:" (hashtags), "r:" (URLs) |
+| contact | pubkeys to whom the user sent DMs or started an interaction such as an one-on-one audio/video call | "p:" |
+| follow | followed pubkeys | "p:" |
+| follow-hashtag | followed hashtags | "t:" |
+| file | adds events and files (urls) to a root directory similar to "/" | "i:", "a:", "r:" (url) |
+| friend | friends' pubkeys | "p:" |
+| mute | pubkeys whose posts the user doesn't want to see in their feeds | "p" (pubkey) |
+| mute-event | muted notes | "i:" (event), "a:" (event) |
+| mute-hashtag | muted hashtags | "t:" (hashtag) |
+| mute-word | muted words | "word:" (lowercase string) |
+
+### User-generated
+
+Set names must be prefixed with a "\`" (backtick) when on `n` tags to avoid name clashing.
+To keep track of these custom names, a "Custom Set" `kind:30061` event is used for each of them.
+The `d` tag is the name. Example:
+
+```js
+{
+  kind: 30061,
```

Comment:
Answered on conversation below but will paste it here:

> About user-generated/custom sets, they need kind:30061 (an alternative to same https://github.com/nostr-protocol/nips/pull/761's kind:10008 concept, but individual events instead of a list) to **keep track of their names** (client can then present the custom names and the standard ones on a "dropdown" when adding something to a set). **There's more to kind:30061 need:** marking a custom set name as "n=follow" will let client know the list may be used as custom feed.